lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,71 @@
+<div class="starbook">
+  <h1>StarBook <small>Mk V</small></h1>
+  <p>with elementary OS</p>
+  <div>
+    <a href="https://en.starlabs.systems/products/starbook/elementary?rfsn=4227837.e8f025" class="button suggested">Buy Now</a>
+  </div>
+
+  <section id="stealth">
+    <img alt="StarBook Mk V, partially closed, from top" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/stealth.png") %>" />
+    <h3>Stealth power.</h3>
+    <p>StarBook Mk V looks unassuming in matte black anodized aluminum, but it’s engineered to pack a punch. 11th-gen Intel Core processors deliver performance on demand while sipping power. Low-latency memory keeps everything snappy. And the over-provisioned solid state StarDrive offers incredible speeds for opening your apps and files.</p>
+  </section>
+
+  <section id="display">
+    <img alt="StarBook Mk V open 180 degrees" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/hinge.png") %>" />
+    <div>
+      <h3>Matte or glossy—why not both?</h3>
+      <p>The 14-inch ARC display features a protective hard coating that lets color shine through like glass while diffusing ambient light like a matte display. Combined with IPS technology and 400 nits of brightness, the display is visible in nearly any light and angle. Which is convenient, because the 180° hinge lets you get some pretty wild angles.</p>
```

Comment:
We've got a lot of vertical space next to this image, probably another good place for more of a list layout like:

**14-inch ARC display with a protective hard coating**
Colors shine through like glass while diffusing ambient light like a matte display

**400 nits of brightness with anti-glare**
Visible in nearly any light. [Maybe some language like working outdoors]

**IPS technology and a 180° hinge**
~Which is convenient, because the 180° hinge lets you get some pretty wild angles.~
[Something about why/when this is practical like sharing your screen with a pair coding buddy or something. I dunno, like some kind of thing a person can say "Oh yeah I can see myself doing that"]